axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceServiceImpl.java

```diff
@@ -854,4 +857,30 @@ public Boolean checkPartnerBankDetailsList(Invoice invoice) {
 
     return true;
   }
+  
+  protected boolean checkEnablePDFGenerationOnVentilation(Invoice invoice) 
+  {
+	  if(appAccountService.getAppInvoice().getIsEnablePDFGenerationForCustomersOnVentilation()) 
+	  {
```

Comment:
Better to use the helper to know if it's a sale or purchase invoice/refund :
https://github.com/axelor/axelor-business-suite/blob/master/axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceToolService.java#L89

Reply:
It's done, check if it's correct.